services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -355,6 +356,46 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to sync meters for unmetered flow"() {
+        given: "A flow with custom meter rate and burst, that differ from defaults"
+        def src = data.srcSwitch
+        def dst = data.dstSwitch
+        assumeTrue("Desired switch combination is not available in current topology", src && dst)
+
+        and: "A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true' fields, that differ from defaults"
+        def flow = flowHelper.randomFlow(src, dst)
+        flow.ignoreBandwidth = true
+        flow.maximumBandwidth = 0
+        flowHelper.addFlow(flow)
+
+        when: "Update burst parameter in meter"
+        northbound.resetMeters(flow.id)
+//        TODO add check response: body or message
+
+        then: "Delete the created flow"
+        flowHelper.deleteFlow(flow.id)
+
+        where:
```

Comment:
1. Every test with 'where' block should have an `@Unroll` annotation (refer to spock docs for more info why)
2. A rule of thumb would be to somehow distinguish 'iterations' by putting some variable in test name (e.g. the `#description`)
3. I don't really think that it worth iterating though all possible switch pairs, since this part is not _that_ switch dependent (since we are not even installing the meter). I think it'd be enough to check on a random OF_13 switch pair.